### ISSUE TITLE
Can you please add back alt+1, alt+2 to jump to the tab?

### ISSUE THREAD

**Issue by jchain** [paolo-sz/fatty#7]
Thank you so much for your non-stop sync up to the upstream mintty these years. I'm stilling using a very early version of your branch where it supports the shortcut alt+1, alt+2, ... to jump to the tab 1, 2, ...

Can you consider adding `alt+1` back to the feature? I personally find it very helpful.

**Comment by paolo-sz** [comment]
Well... I don't remember that feature was ever implemented... and then I do not remember it was ever removed...

Can you please specify which commit are you using?

**Comment by paolo-sz** [comment]
Ok, even if I do not remember that feature was ever implemented, I see its friendliness so I decided to anyway implement it.
The problem is that Alt+\<n\> is already doing something so I moved to Ctrl+\<n\> where n can range from 0 to 9 (0 means the 10th tab).
I also grouped all the windows tab shortcuts into a specific section of the option panel in order to allow the user to enable or disable them as a whole (enabled by default).
